src/components/nav/nav.stories.ts

```diff
@@ -0,0 +1,109 @@
+import { storyFilters } from "../../../.storybook/helpers";
+import readme from "./readme.md";
+import { html } from "../../../support/formatting";
+
+export default {
+  title: "Components/Nav/Nav",
+  parameters: {
+    notes: readme
+  },
+  ...storyFilters()
+};
+
+export const simple = (): string =>
+  html`
+    <calcite-shell>
+      <calcite-nav slot="header">
+        <calcite-menu slot="primary-content-center">
+          <calcite-menu-item text="Example nav item 1" text-enabled></calcite-menu-item>
+          <calcite-menu-item text="Example nav item 2" text-enabled active></calcite-menu-item>
+          <calcite-menu-item text="Example nav item 3" text-enabled></calcite-menu-item>
+        </calcite-menu>
+      </calcite-nav>
+    </calcite-shell>
+  `;
+
+export const primarySlots_TestOnly = (): string =>
+  html`
+    <calcite-shell>
+      <calcite-nav slot="header">
+        <calcite-chip slot="logo"> logo </calcite-chip>
+        <calcite-action slot="nav-action" text="menu" icon="hamburger"> </calcite-action>
+        <calcite-chip slot="primary-content-start"> content </calcite-chip>
+        <calcite-chip slot="primary-content-center"> content </calcite-chip>
+        <calcite-chip slot="primary-content-end"> content </calcite-chip>
+        <calcite-chip slot="user"> user </calcite-chip>
+        <calcite-chip slot="secondary-content-start"> content </calcite-chip>
+        <calcite-chip slot="secondary-content-center"> content </calcite-chip>
+        <calcite-chip slot="tertiary-content-start"> content </calcite-chip>
+        <calcite-chip slot="tertiary-content-center"> content </calcite-chip>
+      </calcite-nav>
+    </calcite-shell>
+  `;
+
+export const primaryAndSecondarySlots_TestOnly = (): string =>
+  html`
+    <calcite-shell>
+      <calcite-nav slot="header">
+        <calcite-chip slot="logo"> logo </calcite-chip>
+        <calcite-action slot="nav-action" text="menu" icon="hamburger"> </calcite-action>
```

Comment:
For these can we just use the `nav-action` prop and not the slot so we can test what the component renders here? 

Or if we want to test the slot on some, just use the Chip as the slot content like the others?

Reply:
there is a test which adds both `nav-action` prop and slot to test out what component renders. I can add a separate one just for the prop. 

thought using `calcite-action` in `nav-action` slot is the recommended practice.

Reply:
It is for that slot in practice, but this is just showing the slot positioning is correct, like we are not placing a `calcite-nav-logo/user` in the `logo/user` slot. For now I think we want to incentivize the property over the slot, especially in a case like this where it's the same as what the component renders. For the purposes of this test it could just be a chip indicating what's there is correct.